### ISSUE TITLE
 How to get vgg_init.pth?

### ISSUE THREAD

**Issue by ucmmesa** [yaoqi-zd/SGAN#9]
Could you please give any advises to get the initial network parameters like the file of vgg_init.pth?Should I train the network from beginning on the dataset of Imagenet to get  vgg_init.pth and then transfer and refine the network?

**Comment by yaoqi-zd** [comment]
@ucmmesa Hi, suppose you want to use a new backbone network, usually this requires three steps:
1. train the network on ImageNet, if the pretrained weights are avaiable, like pretrained resnet, you can skip this step;
2. fine-tune the network on target dataset(e.g. VOC) for multi-label classification task;
3. use the fine-tuned weights to initialize our SGAN network.

**Comment by ucmmesa** [comment]
@yaoqi-zd Very clear.Thanks!
